docs/tutorial/part-one/index.md

```diff
@@ -79,16 +79,17 @@ export default () => (
 
 ![More changes with hot reloading](03-more-hot-reloading.png)
 
-4.  Add an image. (In this case, a random image from Unsplash).
+4.  Add an image. (In this case, the Gatsby astronaut).
 
-```jsx{7}:title=src/pages/index.js
+```jsx{2,8}:title=src/pages/index.js
```

Comment:
Can you explain to me what this change is please? I haven't seen it before...

Reply:
This is code snippet line highlighting. It specifies that line 2 and 8 should be highlighted to denote the changes made.

Reply:
TIL - thank you.